config/_default/params.toml

```diff
@@ -18,7 +18,7 @@ day_night = true
 #   Latest font sets (may require updating): https://sourcethemes.com/academic/themes/
 #   Browse built-in font sets in `themes/academic/data/fonts/`
 #   Browse user installed font sets in `data/fonts/`
-font = ""
+font = "ua_font_set"
```

Comment:
Is this going to be negatively affecting my widget? https://github.com/az-digitalag/group-website/blob/4fe047f6480ccab65cef97c0318b45bf144a388e/layouts/partials/widgets/slider-header.html#L1

Reply:
> Is this going to be negatively affecting my widget? 

Probably not. I think that style will override any global styles. On the other hand, if the Montserrat font is wildly different than the UA font style you might want to remove the `font-family` there and see what your widget looks like with the official 'approved' font. I suspect it will still work OK, and also look more integrated with the rest of the site.

Reply:
I will put an issue in for taking a look after the massive quantity of merges takes place: https://app.zenhub.com/workspaces/ua-ag-data-science-5a57a3198339f11ba1c85775/issues/az-digitalag/organization/263